source/OptimaJet.DWKit.StarterApplication/Controllers/ProjectsController.cs

```diff
@@ -29,28 +29,27 @@ public class ProjectsController : BaseController<Project>
 
         [HttpPost("{id}/token")]
         public async Task<IActionResult> GetProjectToken(int id)
-        {
-            var project = await service.GetAsync(id);
+        {
+            var project = await service.GetAsync(id);
             if (project == null || project.WorkflowProjectUrl == null)
             {
                 return NotFound();
-            }
+            }
 
-            var token = await BuildEngineProjectService.GetProjectTokenAsync(id);
+            var token = await BuildEngineProjectService.GetProjectTokenAsync(id);
             if (token == null || token.SecretAccessKey == null)
             {
                 return NotFound();
-            }
+            }
             var projectToken = new ProjectToken
             {
-                Id = id,
-                SessionToken = token.SessionToken,
-                SecretAccessKey = token.SecretAccessKey,
-                AccessKeyId = token.AccessKeyId,
-                Expiration = token.Expiration,
-                Url = project.WorkflowProjectUrl,
-                Region = token.Region
```

Comment:
Region needs to come back, it's still present on develop

Reply:
@NullVoxPopuli: Very curious why you removed Region from the ProjectToken.  This cause pushing data from SAB to fail (when I pushed this this morning).  This was the only change to the backend.

Reply:
a git conflict / rebase diff told me to? :man_shrugging: